cross/build-rootfs.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [BuildArch]"
+    echo "BuildArch can be: arm, arm64"
+
+    exit 1
+}
+
+__CrossDir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+__InitialDir=$PWD
+__BuildArch=arm
+__UbuntuArch=armhf
+__UbuntuRepo="http://ports.ubuntu.com/"
+__UbuntuPackages="build-essential libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev zlib1g-dev libssl-dev libcurl4-openssl-dev libz-dev libcrypto++-dev"
```

Comment:
Yes I'm definitely open to moving some of these common native scripts into BuildTools. I've not had the time recently but I've been wanting to look closer into the native builds to try and figure out how to set it up in a consistent way to share infrastructure via BuildTools. It would be great if someone wants to pick that up.